components/Cards/index.js

```diff
@@ -17,3 +45,34 @@
 // </div>
 //
 // Create a card for each of the articles and add the card to the DOM.
+
+
+cardContainer = document.querySelector('.cards-container');
+axios
+    .get('https://lambda-times-backend.herokuapp.com/articles')
+    .then( response => {
+        console.log(response.data.articles);
+        response.data.articles.bootstrap.forEach( i => {
+            let newCard = cardMaker(i.headline, i.authorPhoto, i.authorName);
+            cardContainer.appendChild(newCard);
+        })
+        response.data.articles.javascript.forEach( i => {
+            let newCard = cardMaker(i.headline, i.authorPhoto, i.authorName);
+            cardContainer.appendChild(newCard);
+        })
+        response.data.articles.jquery.forEach( i => {
+            let newCard = cardMaker(i.headline, i.authorPhoto, i.authorName);
+            cardContainer.appendChild(newCard);
+        })
+        response.data.articles.node.forEach( i => {
+            let newCard = cardMaker(i.headline, i.authorPhoto, i.authorName);
+            cardContainer.appendChild(newCard);
+        })
+        response.data.articles.technology.forEach( i => {
+            let newCard = cardMaker(i.headline, i.authorPhoto, i.authorName);
+            cardContainer.appendChild(newCard);
+        })
```

Comment:
This definitely does the job but it could be refactored and automated so you don't have to write a manual forEach for all topics in the response.data.articles object

You can take a look at [for..in](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in) loops or [Object.keys](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys)